src/utils/data.js

```diff
@@ -14,78 +14,122 @@
+         src: IBMLogo,
+         alt: "IBM Logo"
+     },
+     {
+         href: "https://www.sap.com/",
+         src: SAPLogo,
+         alt: "SAP Logo"
+     },
+     {
+         href: "https://www.arduino.cc/",
+         src: Arduino,
+         alt: "Arduino Logo"
+     },
+     {
+         href: "https://www.torocloud.com/",
+         src: ToroC,
+         alt: "ToroCloud logo"
+     },
+     {
+         href: "https://eclipsesource.com/",
+         src: EclipseSource,
+         alt: "EclipseSource logo"
+     },
+     {
+         href: "https://www.huawei.com/",
+         src: Huawei,
+         alt: "Huawei logo"
+     },
+     {
+         href: "https://www.acquia.com/products/drupal-cloud/cloud-ide",
+         src: Acquia,
+         alt: "Acquia Logo"
+     }
+ ]
+ 
+ export const products = [
+     {
+         href: "https://os.mbed.com/studio/",
+         src: MbedStudio,
+         thumb: MbedStudioMin,
+         alt: "Mbed Studio by Arm"
+     },
+     {
+        href: "https://www.eclipse.org/emfcloud/#coffeeeditoroverview",
+        src: CoffeeEditor,
+        thumb: CoffeeEditorMin,
+        alt: "Coffee Editor Example by EclipseSource"
     },
     {
-        href: "https://www.ericsson.com",
-        src: EricssonLogo,
-        alt: "Ericsson Logo"
+        href: "https://developers.redhat.com/products/codeready-workspaces/overview",
+        src: CodeReady,
+        thumb: CodeReadyMin,
+        alt: "Red Hat CodeReady Workpaces"
     },
     {
-        href: "https://www.arm.com",
-        src: ARMLogo,
-        alt: "ARM Logo"
+        href: "https://www.logicals.com/en/news/100-logi-cals-makes-engineering-cloud-enabled",
+        src: LogiCloud,
+        thumb: LogiCloudMin,
+        alt: "logi.cloud by logi.cals"
     },
     {
-        href: "https://www.redhat.com",
-        src: RedHatLogo,
-        alt: "RedHat Logo"
+        href: "https://theia-ide.org/docs/blueprint_download",
+        src: BluePrint,
+        thumb: BluePrintMin,
+        alt: "Theia Blueprint"
     },
-    {
-        href: "https://about.google/",
-        src: GoogleLogo,
-        alt: "Google Logo"
-    },
-    {
-        href: "https://www.ibm.com",
-        src: IBMLogo,
-        alt: "IBM Logo"
-    },
-    {
-        href: "https://www.sap.com/",
-        src: SAPLogo,
-        alt: "SAP Logo"
-    },
-    {
-        href: "https://www.arduino.cc/",
-        src: Arduino,
-        alt: "Arduino Logo"
-    },
-    {
-        href: "https://www.torocloud.com/",
-        src: ToroC,
-        alt: "ToroCloud logo"
-    },
-    {
-        href: "https://eclipsesource.com/",
-        src: EclipseSource,
-        alt: "EclipseSource logo"
-    },
-    {
-        href: "https://www.huawei.com/",
-        src: Huawei,
-        alt: "Huawei logo"
-    },
-    {
-        href: "https://www.acquia.com/products/drupal-cloud/cloud-ide",
-        src: Acquia,
-        alt: "Acquia Logo"
-    }
-]
\ No newline at end of file
+ ]
+ 
```

Comment:
For additional information, the formatting of this file is incorrect which therefore results in a `newline` not being present at the end of the file. I won't nitpick for the moment, we can think of adding rules and formatting properly in a subsequent pull-request.

Reply:
#177